community/import-tool/src/docs/ops/import-tool.asciidoc

```diff
@@ -418,3 +459,44 @@ The command line arguments would remain the same as before:
 ----
 include::id-spaces.adoc[]
 ----
+
+=== Missing nodes in relationships file
```

Comment:
`Missing nodes in relationships input data` or something instead, since there may be multiple files providing relationship input data.